qiita_db/artifact.py

```diff
@@ -0,0 +1,598 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+from __future__ import division
+from future.utils import viewitems
+from datetime import datetime
+
+from .base import QiitaObject
+from .sql_connection import TRN
+from .exceptions import (QiitaDBArtifactCreationError,
+                         QiitaDBArtifactDeletionError,
+                         QiitaDBOperationNotPermittedError)
+from .util import (convert_to_id, insert_filepaths,
+                   move_filepaths_to_upload_folder, retrieve_filepaths)
+from .software import Parameters, Command
+from .study import Study
+from .metadata_template import PrepTemplate
+
+
+class Artifact(QiitaObject):
+    r"""A piece of data that different commands can be applied on
```

Comment:
Thanks, changed to `Any kind of file (or group of files) stored in the system and its attributes`as it can potentially represent a group of files, not just a single file.